### PR TITLE
Issue #2532168 by suhel.rangnekar: [D8] Created jqplot chart example …

### PR DESCRIPTION
**Pull request by rangnekarsuhel**
Issue #2532168 by suhel.rangnekar: [D8] Created jqplot chart example SubModule under jqplot

### CODE REVIEW

File: jqplot_example/jqplot_example.module [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?php
```
Comment (sharique) [inline jqplot_example/jqplot_example.module]: Remove empty file.

File: jqplot_example/src/Controller/jqplotExampleController.php [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+   *   A renderable array of line chart and info about libraries files.
+   */
+  public function lineCharts() {
+
+    // Start building the content.
+    $build = array();
+
+    $build['top_content'] = array(
+      '#markup' => <<<INFOMARKUP
+<ol>
```
Comment (sharique) [inline jqplot_example/src/Controller/jqplotExampleController.php]: Indention is not correct, run drupalcs.

File: jqplot_example/src/Controller/jqplotExampleController.php [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+    // Start building the content.
+    $build = array();
+    $build['top_content'] = array(
+      '#markup' => <<<INFOMARKUP
+        <ol>
+        <li>Below is a default bar plot. Bars will highlight on mouseover. Events are triggered when you mouseover a bar and also when you click on a bar. Here We capture the 'jqplotDataClick' event and display the clicked series index, point index and data values. When series data is assigned as a 1-dimensional array as in this example, jqPlot automatically converts it into a 2-dimensional array for plotting. So a series defined as [2, 6, 7, 10] will become [[1,2], [2,6], [3,7], [4,10]].</li>
+        <li>Charts on this page may depend on the following plugins:<br>
+          <p>['#attached']['library'][] = 'jqplot/jqplot.barRenderer'</p>
+          <p>['#attached']['library'][] = 'jqplot/jqplot.pieRenderer'</p>
```
Comment (sharique) [inline jqplot_example/src/Controller/jqplotExampleController.php]: Why adding both bar and pie renderer?

File: jqplot_example/src/Controller/jqplotExampleController.php [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+    // Main container DIV. We give it a unique ID so that the JavaScript can
+    // find it using jQuery.
+    $build['content'] = array(
+      '#markup' => '<div><span>You Clicked: </span><span id="info-bar">Nothing yet</span></div><div id="chart-bar" class="jqplot-target"></div></pre>',
+    );
+
+    // Attach library containing css and js files.
+    $build['#attached']['library'][] = 'jqplot/jqplot.barRenderer';
+    $build['#attached']['library'][] = 'jqplot/jqplot.pieRenderer';
```
Comment (sharique) [inline jqplot_example/src/Controller/jqplotExampleController.php]: Same here.

File: jqplot_example/jqplot_example.links.menu.yml [hunk 5]
```diff
@@ -0,0 +1,28 @@
+jqplot.example:
+  title: jqplot Example
```
Comment (sharique) [inline jqplot_example/jqplot_example.links.menu.yml]: My be we should change it to static example as we are going add dynamic examples.

File: jqplot_example/src/Controller/jqplotExampleController.php (hunk 2 above)
Comment (rangnekarsuhel) [inline jqplot_example/src/Controller/jqplotExampleController.php]: I have set my sublime as per coding stander...but let me check with https://www.drupal.org/project/coder

File: jqplot_example/jqplot_example.links.menu.yml (hunk 5 above)
Comment (rangnekarsuhel) [inline jqplot_example/jqplot_example.links.menu.yml]: Not clear for this point, what you want here...

Comment (sharique) [inline jqplot_example/jqplot_example.links.menu.yml]: We are going to add dynamic examples also, so I think menu should clearly say static/dynamic examples.

File: jqplot_example/jqplot_example.info.yml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+name: jqPlot Charting library Example
+type: module
+description: "jQPlot chart library example module"
+core: 8.x
+version: 8.x-1.x-dev
```
Comment (sharique) [inline jqplot_example/jqplot_example.info.yml]: Remove version.

Comment (rangnekarsuhel) [inline jqplot_example/jqplot_example.info.yml]: @sharique Removed the version.

**Pull request merged**